### ISSUE TITLE
Does sequencing depth matter? 

### ISSUE THREAD

**Issue by fangfang0906** [genome/sciclone#55]
Hi, thank you for developing this tool. My mutation data comes from regular WES (100X coverage). I noticed that other papers recommend deep WES (700-800X coverage). Does the sequencing depth of WES matter?

My second question is about input files. Do you suggest inputting all mutations or only exomic mutations?

My third question is related to the plot below, which I generated by inputting all mutations from regular WES. It seems that these two clusters don't separate well. Could you please help comment on the figure? Thank you in advance!
<img width="895" alt="Screenshot 2023-08-07 at 5 23 40 PM" src="https://github.com/genome/sciclone/assets/30477318/a572b706-252b-4637-951f-febc534aed7d">

**Comment by chrisamiller** [comment]
Yes, sequencing depth matters a great deal. See Figure 5 of this paper for a clear illustration of how increased depth reduces sampling error and increases resolution. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4669575/

Inputting more mutations should generally give better results, assuming that they are high confidence calls.

Assuming no copy number events, I agree that your figure suggests that there are at least two clusters (a founding clone and one or more subclones). Without additional data (more sequencing depth, or more timepoints with clonal shifts) it's difficult to say more than that.

**Comment by fangfang0906** [comment]
Thanks a lot for your reply!
